packages/frontend/src/pages/Token/index.tsx

```diff
@@ -20,6 +20,7 @@ import LiquidityForm from './LaunchForm/Liquidity'
 import TeamAllocationForm from './LaunchForm/TeamAllocation'
 import TokenMetrics from './Metrics'
 import * as styles from './style.css'
+import Communities from './Communities'
```

Comment:
Same here, import should be sorted

Reply:
thanks for the review will implement the corrections